src/vs/editor/contrib/codeAction/browser/codeActionModel.ts

```diff
@@ -72,7 +72,10 @@ class CodeActionOracle extends Disposable {
 		if (selection.isEmpty() && trigger.type === CodeActionTriggerType.Auto) {
 			const { lineNumber, column } = selection.getPosition();
 			const line = model.getLineContent(lineNumber);
-			if (line.length === 0) {
+			if (
+				line.length === 0
+				|| (/^\s*$/g.test(line) && column - 1 === line.length)
```

Comment:
Isn't this duplicating the case a few lines below? 
https://github.com/microsoft/vscode/blob/5f9b8fa76331d69494e5adb75f8be48a9660cc6a/src/vs/editor/contrib/codeAction/browser/codeActionModel.ts#L89

Reply:
It's not the exact same check, but I'll join the checks

Reply:
It's not the exact same check, I will reuse what I can